config/initializers/artsy_eventservice.rb

```diff
@@ -4,7 +4,7 @@
   config.app_name = 'artsy-convection'
   config.event_stream_enabled = true
   config.rabbitmq_url = ENV['RABBITMQ_URL']
-  config.tls = true
+  config.tls = ENV['RABBITMQ_TLS']
```

Comment:
**question:** config type change?

I'm not really familiar with `Artsy::EventService` so I could be off here, but note that this change will change the type being set for this config. What's here now is a `Boolean` and what ENV will return is always a `String` - does that matter?

*format via [Conventional Comments](https://conventionalcomments.org/)*

Reply:
Could this be set by conditionally checking for `RABBITMQ_CA_CERT` / `RABBITMQ_CLIENT_CERT` / `RABBITMQ_CLIENT_KEY`?

Reply:
Thanks @jonallured ! Noob mistake on my part. @izakp - Oh, that's indeed how Gravity does it, it sets `config.tls` boolean by presence of client cert or not. Okay. so I am copying Gravity's approach. Thanks.